tests/components/frigidaire/test_config_flow.py

```diff
@@ -0,0 +1,97 @@
+"""Test the frigidaire config flow."""
+from unittest.mock import patch
+
+from homeassistant import config_entries, setup
+from homeassistant.components.frigidaire.config_flow import CannotConnect, InvalidAuth
+from homeassistant.components.frigidaire.const import DOMAIN
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import RESULT_TYPE_FORM
+
+
+async def test_form(hass: HomeAssistant) -> None:
+    """Test we get the form."""
+    await setup.async_setup_component(hass, "persistent_notification", {})
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": config_entries.SOURCE_USER}
+    )
+    assert result["type"] == "form"
+    assert result["errors"] == {}
+
+    with patch(
+        "homeassistant.components.frigidaire.config_flow.ConfigFlow.async_step_user",
```

Comment:
Please do not mock out core functionality, instead, mock out the client library so the full flow in the core can be tested.